sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -44,6 +44,10 @@ enum { FPM_GET_USE_SOCKET = 1, FPM_STORE_SOCKET = 2, FPM_STORE_USE_SOCKET = 3 };
 static void fpm_sockets_cleanup(int which, void *arg) /* {{{ */
 {
 	unsigned i;
+	unsigned socket_set_count = 0;
+	unsigned socket_set[FPM_ENV_SOCKET_SET_MAX];
+	unsigned socket_set_buf = 0;
+	char envname[16];
```

Comment:
I know that it won't get over 16 in reality but some static analyzers could complain as technically int can have more numbers and `sprintf` below could overflow so I would go with 32.